models/token_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type TokenTestSuite struct {
+	suite.Suite
+}
```

Comment:
But if we have multiple test files, won't there be a name collision if they each have a `TestMain`?